modules/story_factions/readme-original.txt

```diff
@@ -0,0 +1,15 @@
+			  Story Factions by Jason Hood
```

Comment:
I think we can stick with just a credit rather than the full readme, it might be more appropriate to just add him to the credits of the existing readme